bc_obps/bc_obps/settings.py

```diff
@@ -26,7 +26,7 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get("DJANGO_SECRET_KEY")
+SECRET_KEY = os.environ.get("SECRET_KEY")
```

Comment:
did this change again? It was 'SECRET_KEY', and then a PR last week changed it to 'DJANGO_SECRET_KEY` https://github.com/bcgov/cas-registration/pull/234

Reply:
this is why I need rebasing coaching! No, it didn't change again. Git is just playing monkey business and I'm too incompetent to catch them at it